### PR TITLE
PEAR-1441: add tooltip to table search with unique styling

### PR DESCRIPTION
**Pull request by ocshawn**
## Description
- add tooltip to table search with unique styling

## Checklist

- [x] Added proper unit tests
- [x] Left proper TODO messages for any remaining tasks
- [x] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

## Screenshots/Screen Recordings (if Appropriate)
Screenshot from 2023-10-26 17-30-55

### CODE REVIEW

**Comment by ocshawn** [comment]
ready to look at again, please let me know if you find more tables or modals where its not working correctly.

**Comment by wteouchicago** [comment]
I'm seeing the "tooltip" showing up as expected now.

Spotted one thing that we can probably fix later if it's a pain -- see screenshot. The tooltip is appearing over the header of summary pages when the pages are loaded in modals.

Screenshot 2023-11-03 at 12 17 34 AM

**Comment by ocshawn** [comment]
yes it is a known issue, another option would be to go with a fix that does not have any accessibility, as the frameworks need to be updated.

**Comment by cemile-ctds** [comment]
The tooltip text size appears to be set as text-sm, but I couldn't confirm whether something is overwriting this class because it seems larger than the placeholder text.

Additionally, the font-family is currently Montserrat. We should use Noto Sans for tooltips.

**Comment by ocshawn** [comment]
font and font size fixed, good catch

File: packages/portal-proto/src/components/Table/VerticalTable.tsx [hunk 1]
```diff
@@ -169,6 +177,25 @@ function VerticalTable<TData>({
     handleChange({ newSearch: "" });
   };
 
+  const tooltipContainer = search?.tooltip
```
Comment (paribartandhakal) [inline packages/portal-proto/src/components/Table/VerticalTable.tsx:180]: Nitpick: Component name should be PascalCased. 
```suggestion
  const TooltipContainer = search?.tooltip
```

File: packages/portal-proto/src/features/GeneSummary/GeneSummary.tsx [hunk 2]
```diff
@@ -280,6 +280,7 @@ const GeneView = ({
                   cohortFilters={cohortFilters}
                   genomicFilters={genomicFilters}
                   isModal={isModal}
+                  inModal={isModal}
```
Comment (paribartandhakal) [inline packages/portal-proto/src/features/GeneSummary/GeneSummary.tsx:283]: I feel like `isModal` is sufficient and we don't need a new prop`inModal`. Is there any reason why you added this when there is already `isModal`?

Comment (ocshawn) [reply packages/portal-proto/src/features/GeneSummary/GeneSummary.tsx:283]: from what i see `isModal` is for opining links in modals where `inModal` is for identifying if you are currently in a modal

Comment (ocshawn) [reply packages/portal-proto/src/features/GeneSummary/GeneSummary.tsx:283]: this is not the only place that the component is called, in other locations it has separate values

**Pull request merged**